src/figures/figureW.jl

```diff
@@ -61,50 +61,38 @@ function plotCellTypeEffects(wdf, dataType; legend = true)
 end
 
 
-function plotDepletionSynergy(IgGXidx::Int64, IgGYidx::Int64, fit::fitResult; L0, f, murine::Bool, c1q = false, neutralization = false)
+function plotDepletionSynergy(IgGXidx::Int64, IgGYidx::Int64; L0 = 1e-9, f = 4, murine = true, fit = nothing, Cellidx = nothing, c1q = false, neutralization = false, ex = false)
     Xname = murine ? murineIgGFucose[IgGXidx] : humanIgG[IgGXidx]
     Yname = murine ? murineIgGFucose[IgGYidx] : humanIgG[IgGYidx]
-    Kav_df = importKav(; murine = murine, c1q = c1q, retdf = true)
+    Kav_df = importKav(; murine = murine, IgG2bFucose = murine, c1q = c1q, retdf = true)
     Kav = Matrix{Float64}(Kav_df[!, murine ? murineFcgR : humanFcgR])
-    FcExpr = importRtot(; murine = murine)
     ActI = murine ? murineActI : humanActI
-
     nPoints = 100
-    IgGC = zeros(Float64, size(Kav, 1), nPoints)
-
-    IgGC[IgGYidx, :] = range(eps(), eps(); length = nPoints)
-    IgGC[IgGXidx, :] = range(1.0, 1.0; length = nPoints)
-    X1 = polyfc_ActV(L0, KxConst, f, FcExpr, IgGC, Kav, ActI, Mix = false)  # size: celltype * nPoints
-
-    IgGC[IgGXidx, :] = range(eps(), eps(); length = nPoints)
-    IgGC[IgGYidx, :] = range(1.0, 1.0; length = nPoints)
-    X2 = polyfc_ActV(L0, KxConst, f, FcExpr, IgGC, Kav, ActI, Mix = false)  # size: celltype * nPoints
-
-    IgGC[IgGXidx, :] = range(0.0, 1.0; length = nPoints)
-    IgGC[IgGYidx, :] = range(1.0, 0.0; length = nPoints)
-    X = polyfc_ActV(L0, KxConst, f, FcExpr, IgGC, Kav, ActI)  # size: celltype * nPoints
-
-    if c1q
-        X = vcat(X, Kav_df[!, :C1q]' * IgGC)
-        X1 = vcat(X1, Kav_df[!, :C1q]' * IgGC)
-        X2 = vcat(X2, Kav_df[!, :C1q]' * IgGC)
+    if ex
```

Comment:
What do these three options for respectively?

Reply:
@cyrillustan The first, ex, was an example plot on figureB1 that showed only binding given an example FcExpr (ActI should = nothing but that hasn't been working). When Cellidx == nothing, the disease model is being used and a fit is provided. This measures depletion across all cell types. Lastly, if a Cellidx is provided, this is single-cell depletion. (I need to add something in the case that both Cellidx and fit are not provided. This would be a problem).